pkg/model/openstackmodel/servergroup.go

```diff
@@ -141,8 +143,14 @@ func (b *ServerGroupModelBuilder) buildInstances(c *fi.ModelBuilderContext, sg *
 		}
 		c.AddTask(portTask)
 
+		metaWithName := make(map[string]string)
+		for k, v := range igMeta {
+			metaWithName[k] = v
+		}
+		metaWithName[openstack.TagKopsName] = fi.StringValue(instanceName)
 		instanceTask := &openstacktasks.Instance{
 			Name:             instanceName,
+			NamePrefix:       s(groupName),
```

Comment:
If the `NamePrefix` is always the group name, why not changing the field to be called `GroupName`?